src/02-crm/contacts.apib

```diff
@@ -134,6 +134,7 @@ Get details for a single contact.
             + added_at: `2016-02-04T16:44:33+00:00` (string)
             + updated_at: `2016-02-05T16:44:33+00:00` (string)
             + `web_url`: `https://app.teamleader.eu/contact_detail.php?id=cde0bc5f-8602-4e12-b5d3-f03436b54c0d` (string)
+            + vat_number: `BE0899623034` (string, nullable)
```

Comment:
Strong PP: place the new property just before `emails`. It does not really belong after the meta `web_url` property.